pkg/filters/data.go

```diff
@@ -21,39 +20,31 @@ var _ utils.Cloner[*DataFilter] = &DataFilter{}
 
 func NewDataFilter() *DataFilter {
 	return &DataFilter{
-		filters: map[events.ID]map[string]Filter[*StringFilter]{},
+		filters: map[string]Filter[*StringFilter]{},
 		enabled: false,
 	}
 }
 
 // GetEventFilters returns the data filters map for a specific event
 // writing to the map may have unintentional consequences, avoid doing so
-func (af *DataFilter) GetEventFilters(eventID events.ID) map[string]Filter[*StringFilter] {
-	return af.filters[eventID]
+// TODO: encapsulate by replacing this function with "GetFieldFilter(fieldName string) StringFilter"
```

Comment:
If possible, `GetFieldFilter(field int)` should receive an int.

Reply:
I think using a string here will be much more clear...
Anyway, I don't see this part as a bottlenceck, if this is your concern. Thinking about the logic - we still do a map lookup using a string of the data field